src/server/database/migrations/08_initialLivesIndexes.ts

```diff
@@ -0,0 +1,9 @@
+import { DatabaseContext } from "../instance";
+
+export default {
+    identifier: '08_initialLivesIndexes',
+
+    async up({ regular: { db } }: DatabaseContext): Promise<void> {
+        await db.collection('lives').createIndex({ firstName: 1, lastName: 1 }, { unique: true });
+    },
+}
```

Comment:
same comment for EOF here

Reply:
Fixed. Will include in next commit (part 4)